rosbag2_py/test/test_sequential_reader.py

```diff
@@ -0,0 +1,71 @@
+# Copyright 2020 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from pathlib import Path
+
+from common import get_rosbag_options
```

Comment:
```suggestion
from .common import get_rosbag_options
```

Reply:
That was originally the way, but it did not work for the pytests when run using `colcon test`, so I changed it to `common`.
```
[0.808s] 1: _______________ ERROR collecting test/test_sequential_reader.py ________________
[0.808s] 1: ImportError while importing test module '/home/developer/viper/colcon_ws/src/ros2/rosbag2/rosbag2_py/test/test_sequential_reader.py'.
[0.808s] 1: Hint: make sure your test modules/packages have valid Python names.
[0.809s] 1: Traceback:
[0.809s] 1: ../test/test_sequential_reader.py:17: in <module>
[0.809s] 1:     from .common import get_rosbag_options
[0.809s] 1: E   ImportError: attempted relative import with no known parent package
```

Reply:
@mabelzhang This might be fixed by adding an empty ``__init__.py`` to the same directory to turn it into a package.

Reply:
I'm not sure if that would work. That ended up confusing the package search, maybe because there are two `__init__.py` files in the package:
```
[0.614s] 1: ImportError while importing test module '/home/developer/viper/colcon_ws/src/ros2/rosbag2/rosbag2_py/test/test_sequential_reader.py'.
[0.614s] 1: Hint: make sure your test modules/packages have valid Python names.
[0.614s] 1: Traceback:
[0.614s] 1: ../test/test_sequential_reader.py:17: in <module>
[0.614s] 1:     from .common import get_rosbag_options
[0.614s] 1: ../test/common.py:15: in <module>
[0.614s] 1:     import rosbag2_py
[0.614s] 1: ../rosbag2_py/__init__.py:21: in <module>
[0.614s] 1:     from rosbag2_py._reader import \
[0.614s] 1: E   ModuleNotFoundError: No module named 'rosbag2_py._reader'
```

Reply:
@mabelzhang This seems to be some peculiarity of colcon test, let's leave it the way it works then.